src/DBALCompatibility.php

```diff
@@ -32,6 +32,9 @@ public static function notSupportedPlatformException(string $method): Exception
         return new Exception("Method $method is not supported for doctrine platform");
     }
 
+    /**
+     * @psalm-return class-string
```

Comment:
```suggestion
     * @psalm-return class-string<\Doctrine\DBAL\Platforms\SQLitePlatform>
```

Reply:
That does not work, since the classes (return-strings) differ per DBAL version. That is why the compatibility is needed here and the classes are not used directly.